test/wt_byte_epr_test.cpp

```diff
@@ -36,7 +39,20 @@ int_vector<8> wt_byte_epr_test<T>::text{[]()
                                             result.resize(std::rand() % 10000);
                                             for (uint32_t i = 0; i < result.size(); ++i)
                                             {
-                                                result[i] = (std::rand() % 3) + 1; // no 0s allowed. produces 1, 2 or 3.
+                                                result[i] = std::rand() % 4;
```

Comment:
`0` is now allowed?

Reply:
It was always allowed. I have no clue why that wasn't tested